src/CMakeLists.txt

```diff
@@ -212,3 +212,25 @@ if(BUILD_DAEMON)
       $<$<BOOL:${MINGW}>:-static>
   )
 endif()
+
+
+add_library(bitcoin_cli STATIC EXCLUDE_FROM_ALL
+  compat/stdin.cpp
+  rpc/client.cpp
+)
+target_link_libraries(bitcoin_cli
+  PUBLIC
+    univalue
+)
+
+
+# Bitcoin Core RPC client
+if(BUILD_CLI)
+  add_executable(bitcoin-cli bitcoin-cli.cpp)
+  target_link_libraries(bitcoin-cli
+    bitcoin_cli
+    bitcoin_common
+    bitcoin_util
+    libevent::libevent
```

Comment:
> crypto is missing here.

In CMake, the `bitcoin_crypto` object library is not a direct dependency of the `bitcoin-cli` target, rather a dependency of the `bitcoin_util` static library.

> Is it actually unneeded in autotools?

No. It is still needed in Autotools. Otherwise, the `libbitcoin_util.a` will have undefined references (in `src/random.cpp`).